### PR TITLE
CRUD/ clean up

### PR DESCRIPTION
**Pull request by AlexandreBonfim**
Terminado os CRUD e clean up on routes.

Porem, a logica no UserInMeetup tera que ser modificada para seguir as regras de negocio do projeto futuramente.

### CODE REVIEW

File: app/Controllers/Http/TagController.js [hunk 1]
```diff
@@ -11,36 +10,29 @@ class TagController {
...
    * Show a list of all tags.
    * GET tags
-   *
-   * @param {object} ctx
-   * @param {Request} ctx.request
-   * @param {Response} ctx.response
-   * @param {View} ctx.view
    */
-  async index ({ request, response, view }) {
+  async index({ request }) {
```
Comment (pedropaulo15) [inline app/Controllers/Http/TagController.js]: This `request` param is not being used, we can remove that I think.

File: app/Controllers/Http/TagController.js [hunk 2]
```diff
@@ -11,36 +10,29 @@ class TagController {
   /**
    * Show a list of all tags.
```
Comment (pedropaulo15) [inline app/Controllers/Http/TagController.js]: In my opinion we should start having more comments on the controllers, to clarify what they are actually doing, what is the concept of tags? What it suppose to do? what is being used for?

File: app/Controllers/Http/TagController.js (hunk 1 above)
Comment (AlexandreBonfim) [reply app/Controllers/Http/TagController.js]: Yes, request is being used as showed below:

```suggestion
async store({ request }) {
    const data = request.only(["name"]);

    const tag = await Tag.create(data);

    return tag;
  }```

File: app/Controllers/Http/TagController.js (hunk 2 above)
Comment (AlexandreBonfim) [reply app/Controllers/Http/TagController.js]: There is a comment "show a list of tags". I don't see anything that we can add to it.  simple and straight forward. tags are related as preferences os each user eg.: I'm user and would like to follow the tags react, react native, etc...

File: app/Controllers/Http/TagController.js (hunk 1 above)
Comment (pedropaulo15) [reply app/Controllers/Http/TagController.js]: I don't see `request` being used within the `index` function, not the `store`.
```javascript
async index ({ request }) {
  const tags = await Tag.query().fetch();
  return tags;
}
```

File: app/Controllers/Http/TagController.js (hunk 2 above)
Comment (pedropaulo15) [reply app/Controllers/Http/TagController.js]: That is what mean, we could have that explanation in the controller file, specifying what it is, what is used for, when it is used. In that way we don't assume everyone knows what we were thinking when making the change or creating something new.
But again, it's just a suggestion which makes our life much easier in the future.

**Pull request merged**